samples/military_feature_schemas/Fields_Cyberspace.csv

```diff
@@ -3,6 +3,7 @@ identity,Integer,4,Standard Identity,NULLABLE,Identity,,False,,
 context,SmallInteger,2,Context,NULLABLE,Context,,False,,
 symbolset,SmallInteger,2,Symbol Set,NON_NULLABLE,,60,False,,
 symbolentity,Integer,4,Cyberspace Entity,NON_NULLABLE,Cyberspace_Entities,,False,,
+status,SmallInteger,2,Status,NULLABLE,Statuses,,False,,
```

Comment:
I prefer not to revisit this since we have already been through this discussion several times and the decision was #1 (which is why we even have a "status" field that applies to layers like TacticalGraphics). To address this issue, please remove "status" from this feature class since "operationalcondition" field and domains makes slightly more sense.